src/dev/flang/fe/LibraryModule.java

```diff
@@ -2382,7 +2382,7 @@ else if (pos == 0)
             var bb = sourceFileBytes(at);
             var ba = new byte[bb.limit()]; // NYI: Would be better if SourceFile could use bb directly.
             bb.get(0, ba);
-            sf = new SourceFile(Path.of(sourceFileName(at)), ba);
+            sf = new SourceFile(Path.of(name() + ".fum").resolve(Path.of(sourceFileName(at))), ba);
```

Comment:
```suggestion
            sf = new SourceFile(Path.of("module " + name() + ": " + Path.of(sourceFileName(at)).toString()), ba);
```
How about this so it doesn't look like a legal path? 

The output is
```
Original feature declared at module base: String.fz:34:10:
```

Does not seem to be the proper way of using Path though.